clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/client/OpenstackClientProvider.groovy

```diff
@@ -785,6 +787,18 @@ abstract class OpenstackClientProvider {
   }
 
   /**
+   * Returns a list of images.
+   * @param region
+   * @param filters
+   * @return
+   */
+  List<Image> listImages(String region, Map<String, String> filters = null) {
```

Comment:
The openstack4j api allows you to pass a map of filters (i.e. name or other key/value pairs).  Thought it would be better to front the more flexible api option and make filters optional, so you can get both list and list(filters) with one method.